python/paddle/distribution.py

```diff
@@ -201,7 +201,6 @@ class Uniform(Distribution):
           import paddle
           from paddle.distribution import Uniform
 
-          paddle.disable_static()
           # Without broadcasting, a single uniform distribution [3, 4]:
           u1 = Uniform(low=3.0, high=4.0)
           # 2 distributions [1, 3], [2, 4]
```

Comment:
In most cases, no need to use numpy in the sample code.

Reply:
done